httpclientservice/httpClientService.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+
+class ClientService {
+  constructor() {
+    let service = axios.create({
+      headers: {}
```

Comment:
Should we import the API keys and endpoints from the .env as variables and place into the headers? 

```
//after the import axios from `axios` statement

const VACCOVID_API_KEY = process.env.VACCOVID_API_KEY
const VACCOVID_API_ENDPOINT = process.env.VACCOVID_API_ENDPOINT

// then inside the axios.create instance:
let service = axios.create({
  baseURL: VACCOVID_API_ENDPOINT,
  headers: { 
   'x-rapidapi-host': 'vaccovid-coronavirus-vaccine-and-treatment-tracker.p.rapidapi.com',
   'x-rapidapi-key': 'VACCOVID_API_KEY',
   },
})
```

Reply:
No but we should be able to pass them as arguments on instantiation so the constructor can use them.